release notes/v0.3.0.md

```diff
@@ -0,0 +1,64 @@
+xk6-browser v0.3.0 is here! :tada:
+
+This is a minor release with some important bug fixes, a couple of new features, improvements and code refactors!
+
+
+## New features
+
+- Implemented `Browser.on(event)`. ([#268](https://github.com/grafana/xk6-browser/pull/268), [#287](https://github.com/grafana/xk6-browser/pull/287))
+
+  `on('disconnected')` returns a `Promise` you can wait on to get notified when the WebSocket connection to the browser is closed. See [this example](https://github.com/grafana/xk6-browser/blob/v0.3.0/examples/browser_on.js) for details.
+
+  This is the first async method in xk6-browser, using the newly released [event loop support in k6](https://github.com/grafana/k6/pull/2228), and marks the start of our transition to async APIs. :tada:
+
+- Exposed preconfigured mobile device emulation settings. ([#289](https://github.com/grafana/xk6-browser/pull/289))
+
+  We maintain an internal list of popular devices with their viewport size, user agent and scaling factor, which is useful to test how a web site looks and behaves on different devices. See [this example](https://github.com/grafana/xk6-browser/blob/v0.3.0/examples/device_emulation.js) for details.
+
+
+
+## Bugs fixed
+
+- Made concealed elements clickable. ([#264](https://github.com/grafana/xk6-browser/pull/264))
+
+  Previously, interacting with an element obscured by another was failing with `wait for selector did not result in any nodes`. Now we properly detect this scenario and retry the action with different scroll positions. This affects all `Element` and `Frame` `click()`, `dblclick()`, `hover()`, `check()`, `uncheck()` and `tap()` actions.
```

Comment:
Does it make sense to add #264 also as a feature? We didn't have an auto-retry mechanism, and it's one of the most important features of PW.

Reply:
:+1: Sure, go ahead.

Reply:
@imiric, how does it sound?